candidatebot.py

```diff
@@ -92,7 +129,7 @@ def does_page_exist(page_to_query):
   except ValueError, ex:
     print "Couldn't parse JSON:", ex
 
-
+@rate_limited(EDIT_PAGES_PER_SECOND)
 def create_page(person, login_cookies):
   """Create a page if it doesn't exist. If it already exists, just silently
```

Comment:
For now it's outputting "Created a page for X" or "Page already exists at X", but if we remove that later, I agree that we should add something to show the passage of time.